### ISSUE TITLE
Interim 3.0.1 release?

### ISSUE THREAD

**Issue by cincodenada** [adobe/css-tools#30]
Hello! The rewrite looks great, looking forward do it. I'm not sure how big of a request it is to cut a release from the old code, but I'm coming from https://github.com/styled-components/jest-styled-components/pull/356 which is a bug that is blocked by a bug whose fix was merged into the original repo, but never released. That fix and a change to the Travis config are literally the only changes from 3.0.0 and it's a very simple (and tested) fix, so it should be pretty safe: https://github.com/reworkcss/css/commits/master

For expedience's sake would it be possible to release a 3.0.1 version from 434aa1733 to this package while the 4.0 release is being finalized? There's currently no published version of the package with that fix in place. It looks like the release process wasn't terribly complicated - there's no build process or anything, the published package appears to be just the repo as-is, with `test/`, `benchmark/`, and the dotfiles stripped out.

Having a 3.0.1 released (or even a prerelease, if you don't want an official release confusing folks) would help us get unstuck there, and since you're planning on maintaining a fork anyway, it would be nice to be able to depend on a fork that will be updated rather than rolling our own.

Either way, thanks taking this on!

**Comment by holblin** [comment]
The change you describe is already included in version 4.0.0.
I can't release version 3.0.1 of `css` because I'm not the owner, that's why this fork was created.
Releasing a version `3.0.1` of `@adobe/css-tools` will probably not help you since the dependent project will need to make a change (and in that case, why not upgrade to `4.0.0`?)

**Comment by holblin** [comment]
I created a PR for `styled-components/jest-styled-components`, this should solve what you describe. Let me know if there are some issues or if it fixes the problem described.

https://github.com/styled-components/jest-styled-components/pull/416

**Comment by cincodenada** [comment]
Oh, when I opened this issue I hadn't realized that there was a 4.0 release already, I'm not sure how I missed that - in that case, yes, that solves the problem! I was working under the assumption that the 4.0 release was not ready yet, but since it is, that makes this irrelevant. Thanks!
